archives/2021/student-reports/improve-and-maintain-15-activities-Sourabha-G.md

```diff
@@ -0,0 +1,23 @@
+***Google Summer of Code, 2021***
+
+*Name - Sourabha G*
+
+*Email - sourabha.project@gmail.com*
+
+*GitHub username - sourabhaa*
+
+
+**Work Completed:**
+
+
+|   Activity Name   |           Comments           |      Link to work      | Link to Release |
+|-------------------|------------------------------|------------------------|-----------------|
+|JAMath activitty|Changed flow of activity by addding game-design features: <ul><li>A UI that shows the `score`, `highscore` of the player which decides the `win`/`lose`, `replay` and `exit` buttons after completion of each game.</li>Timer of 1 minute per game</li></ul><br><br> Fixed overlapping of numbers <br> enable save and restore game data from `Journal`<br> UI Enhancement: Improved the UI of main, level page and buttons <br>combined event checking for MOUSEBUTTONDOWN of game_over into play<br>Flake8 fixes| <ul><li>[Gameover UI](https://github.com/sugarlabs/jamath-activity/commit/a8779284f84095b3671a273633c2e845964be9d7)</li><li>[Event checking fix](https://github.com/sugarlabs/jamath-activity/commit/4800b9882c3f44122a51e8e7df503da0b27bf7b0)</li><li>[UI Enhancement](https://github.com/sugarlabs/jamath-activity/commit/0cc46911c57d4944ba167c743d5b0f3e826297bc)</li><li>[Flake8 fix](https://github.com/sugarlabs/jamath-activity/commit/79ee1f0a2901195526d8302e94b84e0e4e0e4472)</li><li>[Overlapping digits fix](https://github.com/sugarlabs/jamath-activity/commit/6811a0686abff9096fc909abcb7ea8745ebcc9b4)</li></ul> | [Release v4](https://github.com/sugarlabs/jamath-activity/commit/1d9bbb7c2401f5938ef08be7571536f2b3d03072)|
+
+| Recall activity | Changed flow of activity by addding game-design features: <ul><li>A UI that shows the `score`, `highscore` of the player which decides the `win`/`lose` after completion of each game.</li> <li>10 puzzles per game per game</li><li>Automatic restart the game after Gameover UI</li></ul> UI Enhancement: Changed UI of instruction message in each puzzle.<br> Flake8 fixes<br>PyGIWarning fix<br> Updated Sprites <br> remove unnecessary code and refactor| <ul><li>[Game over UI](https://github.com/sugarlabs/recall/commit/7ac64299554c702f9627e652cb6383cddbd844fb)</li><li>[UI Enhamcement](https://github.com/sugarlabs/recall/commit/201d4031b894c98a96e7f908ab37b67c5c25aebc)</li><li>[Save and restore game data from Journal](https://github.com/sugarlabs/recall/commit/1505dd7b51a9d061a312c3e1d133c745734a212e)</li><li>[PyGIWarning fix](https://github.com/sugarlabs/recall/commit/b9e7e3088b88f52bee2025c2237a215a325eafa9)</li><li>[Update sprites](https://github.com/sugarlabs/recall/commit/2b467d06f4a39fde0c5a8c54eb25be63c502ba2b)</li><li>[Automatic restart previous game](https://github.com/sugarlabs/recall/commit/1e7fbc26a63398b246f978d3f8cce6e3401d6848)</li><li>[Code Optimization](https://github.com/sugarlabs/recall/commit/3abf92a764b5d0bc1b5f57a1f83cf6021c576a6e)</li></ul>| [Release v8](https://github.com/sugarlabs/recall/commit/63078f026b8252f4bcba7a0ac960da0d4f5f106a)|
```

Comment:
Repetition of `per game`. 

`Automatic restart the game after Gameover UI` should be `Automatically restart the game after gameover.`